README.md

```diff
@@ -7,12 +7,15 @@ The Relativity Forms Development Tools repository's purpose is to house tools to
 Currently, this repository holds the EH Fiddle Application in both RAP and Source format.
 
 ### EH Fiddle
+The EH Fiddle Application is a tool which allows rapid testing of Relativity Forms PageInteraction event hander JavaScript without having to create the .NET PageInteraction event handler or even a Relativity Application to host it, yet.
+
 * [EH Fiddle source and README](./Source/EH%20Fiddle/)
 * [EH Fiddle RAP](./ready-to-use/EH_Fiddle.rap)
 
 ### Repository Content (General)
-* [Source](./Source/)
 * [Immediately usable assets](./ready-to-use/)
+* [Relativity Forms Event Handler Public Examples](./examples/)
+* [Source](./Source/)
 
 ## Futures
 In the future, this repository will include a JavaScript library for unit testing Relativity Forms in your own development environments, and potentially also a project template for automation of your Relativity Forms development with NodeJS.
```

Comment:
I know this is old text but would it make sense to be more noncommittal and say something like: In the future, this repository will be enhanced with additional examples and tools. Feedback from the community on what you would like to see here is appreciated.